specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/applicationGateway.json

```diff
@@ -804,13 +804,13 @@
           "$ref": "#/definitions/ApplicationGatewayProbeHealthResponseMatch",
           "description": "Criterion for classifying a healthy probe response."
         },
-        "backendPoolName": {
-          "type": "string",
-          "description": "Name of backend pool of application gateway to which probe request will be sent."
+        "backendAddressPool": {
```

Comment:
breaking change in stable version

Reply:
@yungezz , this should not be breaking for any customer. Reason is that previously, these changes got merged incorrectly before the NRP API changes were checked in - so no customer can use it. The NRP API (the only version) has gone in NRP/124. So, we need to correct the definitions here as per the actual backend API.